src/routes.js

```diff
@@ -299,6 +392,10 @@ const routes = [
               // Data Collectors.
               project.permits(['form.read', 'form.update']),
             form: (form) => form.publishedAt != null
+          },
+          title: {
+            key: 'form',
+            parts: ({ form }) => [form != null ? (form.name || form.xmlFormId) : null]
```

Comment:
This fallback comes up often enough that there's actually a method for it: `form.nameOrId()`.